AEXS/aex-141.md

```diff
@@ -13,21 +13,23 @@ Created: 2021-09-11
 
 ## Abstract
 
-A standard implementation of non-fungible tokens for the Aeternity ecosystem. The design goal of the primary interface is to be as compatible with ERC-721 as possible, so that anyone who can work with ERC-721 can work with this interface. However, where Sophia offers a better way, performance and efficiency should prevail over compatibility.
+A standard implementation of non-fungible tokens for the æternity ecosystem. The design goal of the primary interface is to be as compatible with ERC-721 as possible, so that anyone who can work with ERC-721 can work with this interface. However, where Sophia offers a better way, performance and efficiency should prevail over compatibility.
 
 There is no support for unsafe transactions. Therefore all transactions are ought to be safe.
 
+For minting it is expected that different options will evolve over time. The way minting is done has to be defined via extensions.
```

Comment:
doesn't make sense for me to include like this

Reply:
I don't understand the meaning behind, but if this is something that needs explanation we should re-phrase

Reply:
the meaning is that there can be different types of minting processes like we already see in our recent proposal:
- regular minting
- template minting

and maybe somebody else will come up with other minting proposals in the future

Reply:
removed here: https://github.com/aeternity/AEXs/commit/2820512fba310a10bb3c15bce8b40c9eda016498

discussion about different minting events is already in another discussion to be resolved